assets.md

```diff
@@ -2,29 +2,44 @@
 
 This file is intended to list all the assets controlled by OpenTelemetry.
 
-- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
-    - Used to register service accounts and as a NuGet OpenTelemetry organization e-mail. All .NET SIG maintainers are part of this list. Owned by CNCF.
-    - Admin: [add member](https://lists.cncf.io/g/cncf-opentelemetry-net-maintainers/).
+## CI pipelines
+
 - OpenTelemetry [Azure Pipeline](https://dev.azure.com/opentelemetry/).
     - Free tier is used.
     - Currently used for .NET SDK (.NET SIG maintainers are administrators).
     - Admin: [add administrator](https://dev.azure.com/opentelemetry/pipelines/_settings/)).
+
+## Artifact repositories
+
 - NuGet [OpenTelemetry organization](https://www.nuget.org/organization/OpenTelemetry).
     - All .NET SIG maintainers are administrators of this organization.
     - Organization e-mail (cncf-opentelemetry-net-maintainers@lists.cncf.io) is owned by CNCF.
-- MyGet OpenTelemetryCNCF account is registered under the service account (cncf-opentelemetry-net-maintainers@lists.cncf.io). 
+
+- MyGet OpenTelemetryCNCF account is registered under the service account (cncf-opentelemetry-net-maintainers@lists.cncf.io).
     - Individual accounts of .NET SIG maintainers are administrators of the OpenTelemetry feed.
     - Admin: [manage members](https://www.myget.org/feed/Security/opentelemetry).
-- YouTube channel [OpenTelemetry](https://www.youtube.com/channel/UCHZDBZTIfdy94xMjMKz-_MA/videos). 
-    - Ask any of the following people if you need to manage the feed:
-    - Owners: Amye Scavarda Perrin (CNCF rep), Sergey Kanzhelev
-    - Managers: Austin Parker, Ben Sigelman, Eddy Nakamura, Morgan McLean
+
 - NPM [OpenTelemetry Organization](https://www.npmjs.com/settings/opentelemetry/packages)
     - Ask any of the following people if you need access
     - Owner: Mayur Kale @mayurkale22
     - Admin: Daniel Dyla @dyladan
     - Member: Bogdan Drutu @bogdandrutu
+
+## Communication channels
+
 - [Calendar Google Group](https://groups.google.com/forum/#!forum/opentelemetry-calendar)
     - Owner: Morgan McLean @mtwo
+
+- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
```

Comment:
maybe this should be a bullet point under NuGet

Reply:
We also have one for ruby:

```
- Mailing list cncf-opentelemetry-ruby@lists.cncf.io
    - Used as a service account e-mail for Ruby SIG. All Ruby SIG maintainers [@open-telemetry/ruby-maintainers](https://github.com/orgs/open-telemetry/teams/ruby-maintainers) are moderators of this list. Owned by CNCF.
    - Moderators: [add member](https://lists.cncf.io/g/cncf-opentelemetry-ruby/members).
```

@open-telemetry/ruby-maintainers I see a new member on the maintainer team. @fbogsany @mwear I made you moderators so you should be able to add members. Or just contact me and let know who to add to the list

Reply:
it's a service account. So maybe a separate category